composer.json

```
@@ -88,6 +88,7 @@
         "drupal/link_class": "2.1.0",
         "drupal/linkit": "6.1.6",
         "drupal/masquerade": "2.0.0",
+        "drupal/masquerade_log" : "^1.2",
```

```suggestion
        "drupal/masquerade_log" : "1.2.0",
```
We pin most of our Drupal module dependencies to specific versions in order to keep our Quickstart builds as deterministic as possible.  This tends to make it easier to review changes without having unexpected changes creep in from an un update to an unrelated module, etc.